helpers.php

```diff
@@ -96,6 +96,18 @@ function getOrderFromUniqueId($uniqueId)
     return new Order((int) $results['id_order']);
 }
 
+function getChildOrderFromUniqueId($uniqueId)
+{
+    $results = Db::getInstance()->getRow('SELECT id_order 
+    FROM `' . _DB_PREFIX_ . 'altapay_order` 
+    WHERE unique_id = \'' . $uniqueId . '\'');
```

Comment:
This is potential for SQL injection. use the pSQL method.

Reply:
Escaped the variables using pSQL ✅